okio/api/okio.api

```diff
@@ -800,7 +800,7 @@ public final class okio/Utf8 {
 	public static synthetic fun size$default (Ljava/lang/String;IIILjava/lang/Object;)J
 }
 
-public final class okio/internal/-Utf8 {
+public final class okio/internal/_Utf8Kt {
```

Comment:
Don’t we need a `-` here?

Reply:
I think this is what it was doing before

Reply:
No, the API dump on `3.3.0` gives us

```
public final class okio/internal/_Utf8Kt {
	public static final fun commonAsUtf8ToByteArray (Ljava/lang/String;)[B
	public static final fun commonToUtf8String ([BII)Ljava/lang/String;
	public static synthetic fun commonToUtf8String$default ([BIIILjava/lang/Object;)Ljava/lang/String;
}
```